src/Components/Publishing/Fixtures/Articles.ts

```diff
@@ -591,6 +591,12 @@ export const FeatureArticle: ArticleData = {
   slug: "artsy-editorial-path-winning-art-prize",
 }
 
+export const SponsoredArticle = extend({}, FeatureArticle, {
+  sponsor: {
+    white_logo: "https://artsy-media-uploads.s3.amazonaws.com/F4RVgsSXv3q9Nrt59P8gbA%2Ffullscreen.png"
```

Comment:
I'm kinda feeling like we have this opportunity to redo partner logos now and maybe this format is a little clearer than `partner_logo_fullscreen`. Thoughts? Maybe `light_logo` and `dark_logo` is even more flexible...

Reply:
Yeah light_logo/dark_logo makes sense to me -- are there places where a brand has used a different logo for a header placement versus footer for layout reasons as opposed to color?  For example, some brands have an alternative logo for a small placement versus large.

Reply:
Mm not that I can recall -- we've had:

`partner_fullscreen_header_logo`: to account for dark bgs
`partner_logo`: for navs with white background
`secondary_partner_logo`: used only once afaik here:
![image](https://user-images.githubusercontent.com/2236794/33398394-ced615a6-d51c-11e7-9647-afb2215f0831.png)